src/app/recipe-overview-page/recipe-overview-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RecipeOverviewPageComponent } from './recipe-overview-page.component';
+
+describe('RecipeOverviewPageComponent', () => {
+  let component: RecipeOverviewPageComponent;
+  let fixture: ComponentFixture<RecipeOverviewPageComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ RecipeOverviewPageComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RecipeOverviewPageComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7556402)
```suggestion
  it("should create", () => {
```